src/react-native/07-debugging-devtools/debugging-devtools.md

```diff
@@ -10,39 +10,78 @@
 
 In this section, you will:
 
-- TODO
+- [resource](https://reactnative.dev/docs/debugging)
 
-## Objective 1: TODO
+## Objective: Devtools setup
 
-TODO
+We will be setting up dev tools to help us debug and inspect the UI:
 
-### Concept TODO
+**React Native Debugger**
 
-TODO
+<img alt="Screenshot of react native debugger." src="../../static/img/react-native/07-debugging-devtools/react-native-debugger
+.png"/>
 
-### Setup 1
+**React Devtool**
 
+<img alt="Screenshot of a connected dev tool." src="../../static/img/react-native/07-debugging-devtools/dev-tool-connected.png"/>
+
+### Concept TODO
 TODO
 
-### Verify 1
+### Setup
 
-TODO
+✏️ Update **src/package.json** to be:
 
-### Exercise 1
+@diff ../../../exercises/react-native/06-custom-components/03-solution/package.json ../../../exercises/react-native/07-debugging/01-solution/package.json only
 
-TODO
+Since we have a new package let's run the following command.
 
-### Solution 1
+```shell
+npm i
```

Comment:
You should always install dependencies with `npm install`, not modify the `package.json` and then run install.

Example: https://www.bitovi.com/academy/learn-react/setting-up-your-environment.html#setup-3